packages/web3-eth/src/index.ts

```diff
@@ -15,6 +15,25 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+ * The `web3-eth` package allows you to interact with an Ethereum blockchain and Ethereum smart contracts.
+ *
+ * To use this package standalone and use its methods use:
+ * ```ts
+ * import { getBalance, getBlockByNumber } from 'web3-eth';
+ * ```
+ *
+ * To use this package within the `web3` object use:
+ * ```ts
+ * import { Web3 } from 'web3';
+ *
+ * const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546');
```

Comment:
Do you think we should add a valid public mainnet endpoint in examples? so people copying examples will be facilitated. 
https://github.com/arddluma/awesome-list-rpc-nodes-providers#ethereum

and should add a note that this rpc endpoint not managed by this lib maintainers.

Reply:
if we do this, I think there are other places we should make the change too.

Reply:
There is a feature in Docusaurus to provide executable examples. If we want to provide real network endpoints, then we should use that feature to allow users to run these examples.

Reply:
I agree with @nazarhussain, we should investigate executable examples as this was an initial goal of 4.x's documentation. Either way, we run the risk of the endpoint we use being abused/spammed, so we'd want to be careful what endpoint we use as to not cause burden on someone providing a public endpoint

Reply:
We will use executable examples in web3 tutorials section, and may be can explore to get provider string from user in that case for example to run.